live_pipeline/schema.sql

```diff
@@ -0,0 +1,98 @@
+-- cleanup
+
+DROP TABLE IF EXISTS review;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS game;
+DROP TABLE IF EXISTS developer;
+DROP TABLE IF EXISTS publisher;
+DROP TABLE IF EXISTS platform;
+
+-- tables with no foreign key
+
+CREATE TABLE developer(
+    developer_id INT GENERATED ALWAYS AS IDENTITY,
+    developer_name TEXT NOT NULL,
```

Comment:
Would adding a `UNIQUE` constraint here be good? 

- We want to look into this table and pull out the ID of the developer if a matching developer is present.
- We want to add the developer to the table if not present.
- We don't want duplicate developers in this table. 

This should allow us to use `try-except` blocks when adding data which may be more easier.